app/scripts/services/projects.js

```diff
@@ -23,6 +23,7 @@ angular.module('openshiftConsole')
           return  AuthService
                     .withUser()
                     .then(function() {
+                      AuthorizationService.getProjectRules(projectName);
```

Comment:
should this return a promise we wait for, along with the project fetch, and only when both are resolved do we consider the project loaded?